### ISSUE TITLE
worked with same LSTM for decoding

### ISSUE THREAD

**Issue by wlpdlut** [sanyam5/skip-thoughts#1]
Hello,

       You define two lstm for decoding as following,
    
        self.prev_lstm = nn.LSTM(Encoder.thought_size + self.word_size, self.word_size)
        self.next_lstm = nn.LSTM(Encoder.thought_size + self.word_size, self.word_size)
     
      but in forward, only work with prev_lstm, is it miss writing?
     
       # Supply current "thought" and delayed word embeddings for teacher forcing.
        prev_pred_embds, _ = self.prev_lstm(torch.cat([next_thoughts, delayed_prev_word_embeddings], dim=2))  # (maxlen, batch-1, embd_size)
        next_pred_embds, _ = self.prev_lstm(torch.cat([prev_thoughts, delayed_next_word_embeddings], dim=2))  # (maxlen, batch-1, embd_size)

**Comment by sanyam5** [comment]
Thanks for pointing out! This is indeed a mistake. I'll fix it soon. I am surprised that it still gave pretty decent accuracy.
